pkg/client/cli/cmd_status.go

```diff
@@ -250,3 +257,14 @@ func (cs *userDaemonStatus) WriteTo(out io.Writer) (int64, error) {
 	}
 	return int64(n), nil
 }
+
+func modeToString(mode *manager.Mode) string {
+	switch mode {
+	case manager.Mode_MODE_SINGLE.Enum():
+		return "single-user"
+	case manager.Mode_MODE_TEAM.Enum():
+		return "team"
+	default:
+		return "unknown"
+	}
+}
```

Comment:
This is the second function that creates a string from a `manager.Mode` (the first one is in cmd/traffic/cmd/manager/internal/config/trafficManager.go), and it uses different strings.